priv/test_repo/migrations/20221011201645_install_carbonite_foo.exs

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+
+defmodule Carbonite.TestRepo.Migrations.InstallCarboniteFoo do
+  use Ecto.Migration
+
+  def up do
+    Carbonite.Migrations.up(1, carbonite_prefix: "foo")
+    Carbonite.Migrations.up(2, carbonite_prefix: "foo")
+    Carbonite.Migrations.up(3, carbonite_prefix: "foo")
+    Carbonite.Migrations.up(4, carbonite_prefix: "foo")
+    Carbonite.Migrations.up(5, carbonite_prefix: "foo")
+     end
+
+  def down do
+    Carbonite.Migrations.down(5, carbonite_prefix: "foo")
+    Carbonite.Migrations.down(4, carbonite_prefix: "foo")
+    Carbonite.Migrations.down(3, carbonite_prefix: "foo")
+    Carbonite.Migrations.down(2, carbonite_prefix: "foo")
+    Carbonite.Migrations.down(1, carbonite_prefix: "foo")
+  end
```

Comment:
tracking the rabbits with a trigger actually leads to all tests failing, since then no transactions for this are written.

But since I did not need it for the time being I leave it as an exercise for future generations.